test/src/main/java/io/strimzi/test/k8s/KubeClient.java

```diff
@@ -475,6 +475,14 @@ public Secret getSecret(String secretName) {
         return client.secrets().inNamespace(getNamespace()).withName(secretName).get();
     }
 
+    public Secret getSecret(String secretName, boolean namespaced) {
+        if (namespaced) {
+            return getSecret(secretName);
+        } else {
+            return client.secrets().withName(secretName).get();
+        }
+    }
+
```

Comment:
what's the purpose of this?

Reply:
At the beginning of development of these tests, secrets were not in namespace, but cluster-wide.
Later these tests start to show up in namespaces only, and not cluster-wide as before. 
So when I look for secret, I search in cluster-wide, then in given namespace.

See `getClusterSecret()` below, as you mentioned. They are connected.

Reply:
Are you sure that kube client doesn't use the current active context? Why don't you just change the namespace for client by `.inNamespace()` ?